source/about/deployments-and-editions.rst

```diff
@@ -1,53 +1,58 @@
 
-Mattermost self-hosted 
-~~~~~~~~~~~~~~~~~~~~~~
+We use "Editions" to refer to one of two code bases, either open source or commercial, used to deliver our offerings. We use "Offerings" to refer to sets of features and entitlements available either free or as a paid subscription service. 
 
-Mattermost self-hosted provides Mattermost within IT-controlled private environments in public clouds, including AWS and Azure, as well as on-premises in private clouds and virtual or physical servers.
+Self-Hosted Editions 
+--------------------
 
-Once you’ve downloaded and installed Mattermost within your preferred environment, you have the option to use it as-is, or you can access Mattermost's commercial features by starting a trial or by purchasing a subscription. You can start a 30-day free Enterprise trial via **System Console > Edition and License > Start trial** or request a trial online at https://mattermost.com/trial/.
+Self-hosted editions support deploying Mattermost within IT-controlled private environments in public clouds, including AWS, Azure, GCP and Oracle Cloud, as well as on-premises in private clouds and virtual or physical servers.
 
-Mattermost Cloud
-~~~~~~~~~~~~~~~~
+Mattermost Team Edition 
+~~~~~~~~~~~~~~~~~~~~~~~
 
-Mattermost Cloud delivers Mattermost as software-as-a-service in a highly secure, hosted environment designed for privacy-conscious enterprises and developer collaboration.
+Team Edition is a free-to-use, open source, self-hosted collaboration platform offering all the core productivity benefits of competing SaaS solutions. It deploys as a single Linux binary with MySQL or PostgreSQL under an MIT license.
 
-To get started with a 14-day free trial of Mattermost Cloud online, visit https://customers.mattermost.com/cloud/signup. Signing up requires you to create an account with a business email and specify a domain for the workspace. Once you sign up and create your workspace, a wizard will help you get started completing your profile, inviting team members, and integrating with your favorite developer tools.
+Mattermost Team Edition is also bundled inside of the free Mattermost Enterprise Edition code base, which provides the same funcitionality as Mattermost Team Edition, with the additional benefit of being able to trial as well as upgrade into an expanded set of features available with paid subscription, such as "Mattermost Professional" and "Mattermost Enterprise". 
 
-Mattermost editions
---------------------
+Because of the benefits of Mattermost Enterprise Edition we recommend installing it over Mattermost Team Edition, even if you don’t currently need a subscription, so you'll have the flexibility to trial or enable additional features should you need them. However, if you only want to install software with a fully open source license, then Mattermost Team Edition is the best choice.
```

Comment:
```suggestion
Because of the benefits of Mattermost Enterprise Edition, we recommend installing it instead of Mattermost Team Edition, even if you don’t currently need a subscription, so you'll have the flexibility to trial or enable additional features should you need them. However, if you only want to install software with a fully open source license, then Mattermost Team Edition is the best choice.
```